application/actions/requests_test.go

```diff
@@ -840,6 +841,63 @@ func (as *ActionSuite) Test_RequestActions() {
 	}
 }
 
+func (as *ActionSuite) Test_requestsGet() {
+	f := createFixturesForRequestQuery(as)
+
+	tests := []struct {
+		name       string
+		user       models.User
+		requestID  string
+		wantStatus int
+	}{
+		{
+			name:       "authn error",
+			user:       models.User{},
+			requestID:  f.Requests[0].UUID.String(),
+			wantStatus: http.StatusUnauthorized,
+		},
+		{
+			name:       "authz error",
+			user:       f.Users[1],
+			requestID:  f.Requests[3].UUID.String(),
+			wantStatus: http.StatusNotFound,
+		},
+		{
+			name:       "bad request ID",
+			user:       f.Users[1],
+			requestID:  "1",
+			wantStatus: http.StatusBadRequest,
+		},
+		{
+			name:       "not found",
+			user:       f.Users[1],
+			requestID:  domain.GetUUID().String(),
+			wantStatus: http.StatusNotFound,
```

Comment:
What's the difference between a bad request ID and a request ID that's not found? Is the idea that the bad request ID is of an invalid format/type mismatch to the UUID, and the not found is a UUID that isn't a request ID on record?

Reply:
That's correct